spec/truffle/interop/key_info_spec.rb

```diff
@@ -54,6 +110,12 @@
       end
     end
     
+    it "returns :removable for all keys" do
+      @hash.keys.each do |k|
+        Truffle::Interop.key_info(@hash, k).should include(:removable)
+      end
+    end
```

Comment:
Isn't this duplicated with a spec above?

Reply:
Fixed in #1217.